### PR TITLE
replace covid-19 png image with svg

### PR DESCRIPTION
**Pull request by DilwoarH**
https://trello.com/c/ehl7YMaN/77-update-landing-page-with-svg-campaign-graphic

## What
A version of the landing page campaign graphic that sits in line with the PHE coronavirus branding

## Why
To be shared with the programme team, who need to decide whether we use this version 

## Done when
We have sign off (or not) and are ready to pass to the devs for shipping

## IE8
Screenshot 2020-03-25 at 14 40 22

## Chrome
Screenshot 2020-03-25 at 14 40 48

### Before
Screenshot 2020-03-25 at 15 04 19

### After
Screenshot 2020-03-25 at 15 24 28

### CODE REVIEW

File: app/assets/images/corona-landing-page-logos.svg [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<svg width="476" height="61" viewBox="0 0 476 61" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (andysellick) [inline app/assets/images/corona-landing-page-logos.svg]: Just out of curiousity how big is this SVG? The PNG was hi res and came in at 9.17 KB.

Comment (DilwoarH) [reply app/assets/images/corona-landing-page-logos.svg]: > SVG doesn't work in IE8. Not sure if that's a problem or worth putting some effort into a fallback.

Good point - I'll look into that.



> Just out of curiousity how big is this SVG? The PNG was hi res and came in at 9.17 KB.

I think there's just more code in the vector version

**Comment by DilwoarH** [comment]
Waiting on confirmation from Sam

**Comment by DilwoarH** [comment]
@andysellick - I've added a fallback for browsers which don't support SVG

File: app/views/coronavirus_landing_page/_nhs_logo.html.erb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <title>NHS</title>
+    <style>
+        .st0 {
+            fill: #005EB8;
+        }
+        
+        .st1 {
+            fill: #FFFFFF;
+        }
+    </style>
```
Comment (andysellick) [inline app/views/coronavirus_landing_page/_nhs_logo.html.erb]: Is it worth compressing this at all?

File: app/views/coronavirus_landing_page/_nhs_logo.html.erb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    <path class="st0" d="M0 0h69.2v28H0z" />
+    <path class="st1" d="M6.8 2.7h7.5l4.6 15.6h.1l3.1-15.6h5.7L23 25.2h-7.4L10.8 9.6h-.1L7.6 25.2H2L6.8 2.7zM30 2.7h6l-1.7 8.6h7.1l1.8-8.6h6l-4.7 22.5h-6l2-9.6h-7.2l-2 9.6h-6L30 2.7zM65.1 7.8c-1.6-.7-3.3-1.1-5-1-2.4 0-4.3.3-4.3 2.2 0 3.2 8.8 2 8.8 8.9 0 6.2-5.8 7.9-11 7.9-2.4-.1-4.7-.4-7-1.2L48 20c1.7.9 3.6 1.3 5.5 1.3s4.8-.4 4.8-2.7c0-3.6-8.8-2.3-8.8-8.6 0-5.8 5.1-7.6 10-7.6 2.3-.1 4.7.2 6.9 1l-1.3 4.4z" />
+    <%= tag.image(src: image_url("nhs-logo.png"),
+        alt: "NHS logo",
+        "xlink:href": "") %>
```
Comment (andysellick) [inline app/views/coronavirus_landing_page/_nhs_logo.html.erb]: This looks important but also potentially easily overlooked. Is this documented in the commit message or somewhere?

File: app/views/coronavirus_landing_page/_nhs_logo.html.erb (hunk 2 above)
Comment (DilwoarH) [reply app/views/coronavirus_landing_page/_nhs_logo.html.erb]: Done

File: app/views/coronavirus_landing_page/_nhs_logo.html.erb (hunk 3 above)
Comment (DilwoarH) [reply app/views/coronavirus_landing_page/_nhs_logo.html.erb]: I've updated the commit message with a reference and explaination

**Pull request merged**